docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb

```diff
@@ -0,0 +1,467 @@
+    "  * [Prepare a HyP3 InSAR Stack for MintPy](https://nbviewer.org/github/ASFOpenSARlab/opensarlab-notebooks/blob/master/SAR_Training/English/Master/Prepare_HyP3_InSAR_Stack_for_MintPy.ipynb)\n",
+    "  * [MintPy Time-series Analysis](https://nbviewer.org/github/ASFOpenSARlab/opensarlab-notebooks/blob/master/SAR_Training/English/Master/MintPy_Time_Series_From_Prepared_Data_Stack.ipynb)\n",
+    "  \n",
+    "    Note: While these notebooks make some assumptions you're working in OpenSARlab, you can run these \n",
+    "    notebooks outside OpenSARlab by creating [this conda environment](https://github.com/ASFOpenSARlab/opensarlab-envs/blob/main/Environment_Configs/insar_analysis_env.yml)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0a6e353",
+   "metadata": {},
+   "source": [
+    "## 0. Initial Setup\n",
+    "\n",
+    "To run this notebook, you'll need a conda environment with the required dependencies. You can set up a new environment (recommended) and run the jupyter server like:\n",
+    "```shell\n",
+    "conda create -n hyp3-mintpy python=3.10 asf_search hyp3_sdk \"mintpy>=1.5.2\" pandas jupyter ipympl\n",
+    "\n",
+    "conda activate hyp3-mintpy\n",
+    "jupyter notebook hyp3_insar_stack_for_ts_analysis.ipynb\n",
+    "```\n",
+    "Or, install these dependencies into your own environment:\n",
+    "```shell\n",
+    "conda install hyp3-mintpy python=3.10 asf_search hyp3_sdk \"mintpy>=1.5.2\" pandas jupyter ipympl\n",
+    "\n",
+    "jupyter notebook hyp3_insar_stack_for_ts_analysis.ipynb\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64e566f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from dateutil.parser import parse as parse_date\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33543149",
+   "metadata": {},
+   "source": [
+    "### Set parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7fa17bd0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "project_name = '2014_mount_edgecumbe'\n",
+    "work_dir = Path.cwd() / project_name\n",
+    "data_dir = work_dir / 'data'\n",
+    "\n",
+    "stack_start = parse_date('2014-05-01 00:00:00Z')\n",
+    "stack_end = parse_date('2015-05-01 00:00:00Z')\n",
+    "max_temporal_baseline =  37 #days\n",
+    "\n",
+    "data_dir.mkdir(parents=True, exist_ok=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e62c544",
+   "metadata": {},
+   "source": [
+    "## 1. Select InSAR pairs with ASF Search"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4498c66c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import asf_search as asf\n",
+    "import pandas as pd\n",
+    "\n",
+    "search_results = asf.search(\n",
```

Comment:
Should we also filter by VV polarization? (See the search filters used for the burst stack linked from https://asfdaac.atlassian.net/browse/TOOL-899)

Reply:
Ok. I can add the polarization filter.